data/json/overmap/overmap_terrain/overmap_terrain_alien.json

```diff
@@ -34,5 +34,113 @@
     "color": "light_green",
     "see_cost": 2,
     "flags": [ "KNOWN_DOWN", "NO_ROTATE" ]
+  },
+  {
+    "type": "overmap_terrain",
+    "id": "exodii_base_x0y0z0",
+    "name": "rocky butte",
```

Comment:
Could you please describe what's a butte?

Reply:
A butte is a large, jutting rocky hill similar to a mesa but usually with a smaller top